introduction.md

```diff
@@ -0,0 +1,7 @@
+Lorem ipsum dolor sit amet, aeque elaboraret efficiantur an pri. Cibo aliquando eu ius, has augue inermis ut. Omnes putant ius in. Ad movet quaerendum his, vidit essent eam eu.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.